lib/routes/cdn.js

```diff
@@ -99,6 +100,19 @@ module.exports =
           const wrhsAssets = await fastify.tarballToAssets(req.raw);
           const { files, metadata: metas } = wrhsAssets;
           await fastify.uploadAssetFilesToStorage(files, expTimestamp);
+
+          if (fastify.log.security) {
+            fastify.log.security({
```

Comment:
this works, but you could also just do this as events that the internal wrappers can listen to

Reply:
Certainly a way I'll consider in a fast follow refactoring.